core/camel-componentdsl/src/test/java/org/apache/camel/builder/component/ComponentsBuilderFactoryTest.java

```diff
@@ -46,32 +46,32 @@ public void testNegativeDelay() throws Exception {
     public void testIfResolvePlaceholdersCorrectly() {
         context.getPropertiesComponent().setLocation("classpath:application.properties");
 
+        final KafkaConfiguration kafkaConfiguration = new KafkaConfiguration();
```

Comment:
Can you add a TODO here as we should get the component dsl improved so we can do as before (there is a JIRA about it)

Reply:
Its https://issues.apache.org/jira/browse/CAMEL-14568

Reply:
done!